addon/components/tether-tooltip-and-popover.js

```diff
@@ -216,7 +216,7 @@ export default EmberTetherComponent.extend({
       return null;
     }
 
-    const parentElement = $element.parent();
+    const parentElement = $element.parent().parent();
```

Comment:
I think this will break the use when a `tether-tooltip-and-popover` is used by itself. 

```
<div>
  hover here
  {{tether-tooltip-and-popover text="foo"}}
</div>
```